docs/runtimes/functions.md

```diff
@@ -58,6 +58,92 @@ handle the orchestration and monitoring of the distributed task.
 
 <img src="../_static/images/runtime-scaling.png" alt="runtime-scaling" width="400"/>
 
+## Applying storage configurations to functions
+
+In the vast majority of cases, an MLRun function would require access to storage. This storage
+may be used to provide inputs to the function such as data-sets to process or data-streams that contain input events.
+Storage would usually also be used to store function outputs and result artifacts, such as trained models or processed
+data-sets.
+
+As MLRun function may be distributed and executed in Kubernetes pods, the storage used would in most cases be a shared
+storage, and execution pods would need some configurations applied to them to allow the function code access to the 
+storage in question. These configurations may be k8s volume mounts, specific environment variables that contain
+configuration and credentials and so on. Note that these storage configurations are not applicable to functions running
```

Comment:
Not sure what the "so on" means. Generally speaking you should continue the sentence with something like "and other configuration or security settings". This may not be what you mean by "son on" but try not to use the so on.